Cargo.toml

```diff
@@ -12,11 +12,12 @@ description = "mDNS Service Discovery library with no async runtime dependency"
 
 [features]
 async = ["flume/async"]
-default = ["async"]
+default = ["async", "logging"]
```

Comment:
Could you please move `default` line towards after `logging` since `default` is special and I think it's better to be the last line of this section.